src/app/users/users.service.ts

```diff
@@ -73,33 +75,42 @@ export class UsersService {
         switchMap(userIdentity => this.httpClient.put<UserPermission[]>(
           // TODO: Update permissions for given scope via specific API call, e.g.
           // PUT /users/{userId}/permissions/{scope}
-          this.userPermissionsUrlPattern.replace('{userId}', userIdentity.userId), userPermissions)
+          this.personalPermissionsUrlPattern.replace('{userId}', userIdentity.userId), userPermissions)
         )
       );
   }
 
-  getPermissionOwners(scope: UserPermissionScope): Observable<User[]> {
+  getPersonalPermissionOwnersByScope(scope: UserPermissionScope): Observable<User[]> {
     return this.userIdentityService.getUserIdentity()
       .pipe(
         switchMap(userIdentity => this.httpClient.get<UserPermission[]>(
-          // TODO: Request permissions for given scope via specific API call, e.g.
-          // GET /users/{userId}/permissions/{scope}
-          this.userInboundPermissionsUrlPattern.replace('{userId}', userIdentity.userId))
+          this.personalInboundPermissionsByScopeUrlPattern.replace('{userId}', userIdentity.userId).replace('{scope}', scope),
```

Comment:
Since there is no more the `{scope}` part in the URI path and the scope parameter is passed via the query parameter I suggest to remove the call `replace('{scope}', scope)` from here.

Reply:
Thank you. Removed